testu01-stdin/TestU01_stdin.c

```diff
@@ -0,0 +1,211 @@
+/*
```

Comment:
I wasn't really sure where to put this ... opted for a new folder `testu01-stdin`, but no strong opinion on location / naming.

Reply:
Fine with me